.github/workflows/test-integration-postgres-redis-rabbit.yml

```diff
@@ -17,6 +17,11 @@ on:
         default: true
         required: false
         type: boolean
+      global_postgres_ref:
+        description: 'Branch of Postgres @openphone/database repo to use'
+        default: 'master'
+        required: false
+        type: string
```

Comment:
Was this also added by mistake? I don't see it referenced in the workflow.

Reply:
Ah, nvm. I had to load more of the file cause GH only gives the diffs.

Reply:
Yes both inputs are referenced below